### DISCUSSION TITLE
Curious about developing Censys Search Queries?

### DISCUSSION THREAD

**Discussion by thehappydinoa** [thehappydinoa/awesome-censys-queries#19] [General]
Please reach out either on this thread or on Twitter! I will gladly help you fingerprint servers and systems!

**Comment by bleachcow** [comment]
hello this query will show lots of unsecured rtsp streams Sometimes the user changes the password but usually they are wide open with no authentication these are viewable with vlc player.

```dsl
services.software.uniform_resource_identifier: `cpe:2.3:a:realnetworks:realserver:1.0:*:*:*:*:*:*:*`
```

you only have to add a /1 at the end of the link for example "rtsp://xxx.xxx.xxx.xxx/1"

**Reply by thehappydinoa** [reply]
Hey there, thats a great query, would you be able to add it to the README?
